### PR TITLE
system is using redis

### PR DESCRIPTION
**Pull request by zalun**
to test on local system please set the REDISCLOUD_URL environment variable.
in my case redis://localhost:6379

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,3 +12,6 @@ deploy:
     repo: marco-c/mercurius
     branch: master
     node: '4'
+env:
+  REDISCLOUD_URL: redis://localhost:6379
+before_script: sudo redis-server /etc/redis/redis.conf --port 6379
```
Comment (marco-c) [inline .travis.yml]: Is `sudo` needed?

File: index.js [hunk 2]
```diff
@@ -35,58 +39,66 @@ if (!fs.existsSync('./dist')) {
...
-  delete registrations[req.body.token];
-  res.sendStatus(200);
+  var token = req.body.token;
+  client.exists(token, function(err, result) {
+    if (!result) {
+      res.sendStatus(404);
+      return;
+    }
+    client.del(token);
+    client.srem('registrations', token);
```
Comment (marco-c) [inline index.js]: Are these synchronous operations?

File: .travis.yml (hunk 1 above)
Comment (zalun) [inline .travis.yml]: It doesn't even work - I'm looking to make the redis pass tests on travis

File: index.js (hunk 2 above)
Comment (zalun) [inline index.js]: del is there only because we're using the same database as the site (which is wrong)

File: test/test.js [hunk 3]
```diff
@@ -122,4 +127,15 @@ describe('mercurius', function() {
       })
       .expect(404, done);
   });
+
+  it('redis and arrays of hashes', function(done) {
```
Comment (marco-c) [inline test/test.js]: Can you remove this test now?

File: index.js [hunk 4]
```diff
@@ -35,58 +39,66 @@ if (!fs.existsSync('./dist')) {
...
 app.post('/unregister', function(req, res) {
-  if (!registrations[req.body.token]) {
-    res.sendStatus(404);
-    return;
-  }
-
-  delete registrations[req.body.token];
-  res.sendStatus(200);
+  var token = req.body.token;
+  client.exists(token, function(err, result) {
```
Comment (marco-c) [inline index.js:58]: We should add error checking in a follow-up, filed #15.

**Pull request merged**